ngraph/src/ngraph/op/util/op_types.hpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/ngraph_visibility.hpp"
+#include "ngraph/node.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace util
```

Comment:
I think all those functions should take a `shared_ptr` to avoid the need of an additional `.get()` call at the call-site. We can obviously have 2 overloads, one with `Node*` and one with `shared_ptr` per each function.

Reply:
Ok, I will add methods for both ptrs. because we have places where we have a raw pointer or reference to object.